### PR TITLE
Added stable version config, generate pws

### PR DESCRIPTION
**Pull request by open-schnick**

### CODE REVIEW

File: lib/config.shlib [hunk 1]
```diff
@@ -12,9 +12,9 @@ read() { # path, key -> value
 }
 
 delete() { # path, key
-  test -f "$1" && sed -i "" -e "/^$(echo $2 | sed_escape).*$/d" "$1"
-  # UNIX VERSION:
-  # test -f "$1" && sed -i "/^$(echo $2 | sed_escape).*$/d" "$1"
+  test -f "$1" && sed -i "/^$(echo $2 | sed_escape).*$/d" "$1"
+  # MacOs
```
Comment (qvalentin) [inline lib/config.shlib:16]: https://stackoverflow.com/questions/394230/how-to-detect-the-os-from-a-bash-script

Vllt einfach abfragen, ob es macos ist?

File: lib/dockertags.shlib [hunk 2]
```diff
@@ -0,0 +1,12 @@
+getTagsByName() {
+  image="$1"
+  # REMEMBER wget is not native on mac os.
+  tags=`wget -q https://registry.hub.docker.com/v1/repositories/${image}/tags -O -  | sed -e 's/[][]//g' -e 's/"//g' -e 's/ //g' | tr '}' '\n'  | awk -F: '{print $3}'`
```
Comment (qvalentin) [inline lib/dockertags.shlib:4]: why not use curl instead of wget?

File: init_setup.sh [hunk 3]
```diff
@@ -111,15 +131,15 @@ docker create \
 -e SPRING_PROFILES_ACTIVE="prod" \
 -p $rest_port:8080 \
 --network $networkname \
---name $restname filefighter/rest:latest >/dev/null 2>&1
+--name $restname filefighter/rest:$restVersion >/dev/null 2>&1
```
Comment (qvalentin) [inline init_setup.sh:134]: using >/dev/null 2>&1 hides all the download times, would be nice to get some feedback that it is doing something.

**Review by qvalentin — APPROVED** [review]
nice, see comments for improvement ideas

File: lib/dockertags.shlib (hunk 2 above)
Comment (open-schnick) [reply lib/dockertags.shlib:4]: easier to use :D

File: init_setup.sh (hunk 3 above)
Comment (open-schnick) [reply init_setup.sh:134]: maybe there will be a config key in the future something like `show_debug` or `silent_mode`

**Pull request merged**